interface/reports/appt_encounter_report.php

```diff
@@ -512,7 +660,7 @@ function endDoctor(&$docrow)
             <?php echo text($row['pid']); ?>&nbsp;
          </td>
          <td align='right'>
-            <?php echo text($encounter); ?>&nbsp;
+             <a href='javascript;' data-encounter='<?php echo $encounter; ?>' class='goto-encounter'><?php echo text($encounter); ?></a>&nbsp;
```

Comment:
The link to the encounter does not do anything on my system.

Uncaught TypeError: Cannot read property 'selectedEncounterID' of null
    at setEncounter (tabs_view_model.js?v=41:163)
    at chooseEncounterEvent (tabs_view_model.js?v=41:168)
    at HTMLAnchorElement.<anonymous> (appt_encounter_report.php:58)
    at HTMLAnchorElement.dispatch (index.js?v=41:3)
    at HTMLAnchorElement.q.handle (index.js?v=41:3)
setEncounter @ tabs_view_model.js?v=41:163
chooseEncounterEvent @ tabs_view_model.js?v=41:168
(anonymous) @ appt_encounter_report.php:58
dispatch @ index.js?v=41:3
q.handle @ index.js?v=41:3